### ISSUE TITLE
Update CII badge after adding data validation tests

### ISSUE THREAD

**Issue by e0** [MetabolicAtlas/MetabolicAtlas#851]
As [mentioned here](https://github.com/MetabolicAtlas/MetabolicAtlas/issues/780#issuecomment-1014258352), the badge should be updated once #830 is merged.

**Comment by inghylt** [comment]
https://bestpractices.coreinfrastructure.org/en/projects/4276

This is the link to out CII badge. You log in with your github account. To edit the relevant sections, press "Edit" in the top of the page and then fill in the information as you see fit. Feel free to reach out to the team for feedback or questions

**Comment by e0** [comment]
I have updated two of the checklist items for the CII badge

![image.png](https://images.zenhubusercontent.com/60992f121594d60ae1132a98/0a620b9b-6aa7-4e82-bcd9-c918e951268b)

**Comment by e0** [comment]
@inghylt @MalinAhlberg @nanjiangshu Please take a look at this section (either in the screenshot from previous message or directly at: https://bestpractices.coreinfrastructure.org/en/projects/4276#test) and let me know if you think it looks good. We can close this after getting two approvals.

**Comment by inghylt** [comment]
"_The project MUST clearly show or document how to run the test suite(s) (e.g., via a continuous integration (CI) script or via documentation in files such as BUILD.md, README.md, or CONTRIBUTING.md)_." Do we really fulfill that? There is no info in README.md or CONTRIBUTING.md

**Comment by mihai-sysbio** [comment]
I second the more thorough approach proposed by @inghylt above. I know it's not part of the original issue description, but without a new paragraph somewhere it feels like we are treating this too lightly. How about introducing a new section in `CONTRIBUTING.md`? I'm all for keep things short since the test suite is still in the growth phase.

**Comment by e0** [comment]
> "_The project MUST clearly show or document how to run the test suite(s) (e.g., via a continuous integration (CI) script or via documentation in files such as BUILD.md, README.md, or CONTRIBUTING.md)_." Do we really fulfill that? There is no info in neither README.md or CONTRIBUTING.md

Ah nice catch! I mistakenly assumed it was already in the README.md. Should we create a new PR to address this? I was thinking it could go under the list of CLI commands in README.

**Comment by inghylt** [comment]
I also this it makes sense to have it in the list of helper commands. When adding info about how to run tests, maybe we should add the commands regarding linting/formating as well?

**Comment by mihai-sysbio** [comment]
Whatever text we have on the CII badge, it must not contain any new information. So I see a mention of Jest there, and I'd like to make sure that our project contains this text somewhere.

**Comment by e0** [comment]
> Whatever text we have on the CII badge, it must not contain any new information. So I see a mention of Jest there, and I'd like to make sure that our project contains this text somewhere.

Can you please elaborate a bit more on this? Do you mean that we should mention Jest somewhere in the documentation?

**Comment by nanjiangshu** [comment]
Looks good to me except for the point Ingrid mentioned. I think README.md might be a better place for instructions to testing than `CONTRIBUTING.md`.

**Comment by mihai-sysbio** [comment]
> Can you please elaborate a bit more on this? Do you mean that we should mention Jest somewhere in the documentation?

As a general principle, I see this as an implementation of DRY. When describing the project, it's better to have different external sources (be it the CII badge or something else) point to a page that is fully under our "umbrella", be it the website or the repository.

In this particular case, if we go into the details of mentioning Jest, I'd rather have that info as much as possible on some of "our pages", and just refer to these wherever needed, be it on the CII website or even in our Readme.

**Comment by e0** [comment]
> > Can you please elaborate a bit more on this? Do you mean that we should mention Jest somewhere in the documentation?
> 
> As a general principle, I see this as an implementation of DRY. When describing the project, it's better to have different external sources (be it the CII badge or something else) point to a page that is fully under our "umbrella", be it the website or the repository.
> 
> In this particular case, if we go into the details of mentioning Jest, I'd rather have that info as much as possible in one of "our pages", and just refer to these wherever needed, be it on the CII website or even in our Readme.

Ah I see. I was trying to follow the style of other sections on the BadgeApp where we have previously left some relevant information. If you click the "Show details" button for this section they had some examples of commands for other popular tools, so I wanted to make it easy for whoever is reviewing this page to have the information there. Maybe we can discuss more about this in a meeting.

**Comment by e0** [comment]
Added a PR with quite light additions for now so we can continue the discussion there if necessary. Thanks again for all the reviews and feedback!
